libs/ledger-live-common/src/families/celo/api/hubble.ts

```diff
@@ -74,12 +110,24 @@ export const getAccountDetails = async (address: string, accountId: string) => {
   const accountDetails = await fetchAccountDetails(address);
   const spendableBalance = new BigNumber(accountDetails.gold_balance);
   const lockedBalance = new BigNumber(accountDetails.total_locked_gold);
+  const nonvotingLockedBalance = new BigNumber(
+    accountDetails.total_nonvoting_locked_gold
+  );
   const balance = spendableBalance.plus(lockedBalance);
   const indexerStatus = await fetchStatus();
 
-  const allTransactions = accountDetails.internal_transfers.concat(
-    accountDetails.transactions
-  );
+  //TODO: refactor, cache, move to sdk
```

Comment:
indexer returns some of transactions doubled, I have to clean it up using lockedGold contract address. I'll need to refactor this part and cache it

Reply:
Is this a bug in the indexers, or is there a reason transactions are doubled?

Reply:
That's how indexer returns them because that's how Celo works - signing a contract is one transaction, but also an amount is being sent to that contract at the same time which is also caught by an indexer. In LL we care only for a contract transaction so I think it's fine to filter it